branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
     	}
-    	System.out.println("IN AMETHOD"); 
+    	System.out.println("IN AMETHOD");
+    	String password="XXXX";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "password" local variable. ([squid:S1481](http://localhost:9721/coding_rules#rule_key=squid%3AS1481))

[See it in SonarQube](http://localhost:9721/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=2&issues=AWU6DJQJcMk10FtshGW8&open=AWU6DJQJcMk10FtshGW8)

[ISSUE_KEY]: AWU6DJQJcMk10FtshGW8